sql/core/src/main/scala/org/apache/spark/sql/SparkSession.scala

```diff
@@ -1074,6 +1058,24 @@ object SparkSession extends Logging {
       throw new IllegalStateException("No active or default Spark session found")))
   }
 
+  def applyModifiableSettings(
```

Comment:
Can we also add some docs that this is being invoked from PySpark, and that's why it exists in SparkSession object?